src/System.Management.Automation/engine/remoting/fanin/WSManNativeAPI.cs

```diff
@@ -1121,7 +1121,7 @@ internal WSManOptionSet(WSManOption[] options)
                     // Look at the structure of native WSManOptionSet.. Options is a pointer..
                     // In C-Style array individual elements are continuous..so I am building
                     // continuous array elements here.
-                    Marshal.StructureToPtr(options[index], (IntPtr)(_optionSet.options.ToInt64() + (sizeOfOption * index)), false);
+                    Marshal.StructureToPtr(options[index], _optionSet.options + (sizeOfOption * index), false);
```

Comment:
Don't change remoting, especially WSMan

Reply:
The original code results in overhead on 32 bit arch, see: (x86 vs x64)

https://sharplab.io/#v2:EYLgdgpgLgZgHgGiiAhgZwLYB8ACAmARgFgAoHAZgAJ9KBhSgb1MpcoAcAnASwDcUoIlAJJgoABSgd2kygF5KkAO7DREjgAoAlAG5mrCivEyAsuq6jKAexgw00TYz2t9AdkrqRRjpvVtJAOgAVS08ANgAWLUoAaisbOygdJ0oAX1IUoA

Reply:
Any code change that I don't immediately see as functionally equivalent in WSMan or any other high-risk areas of the code cleanup PR will not be accepted.

If this is a bug, open a separate PR.  Do not fix two issues in the same PR.  If it is a bug in WSMan remoting, the change might need to be backported and code cleanup changes will NOT be backported.  So, mixing the two changes makes my job **VERY** difficult.  

If you find a functional issue during a code cleanup PR, please always open a separate PR.